modules/ROOT/pages/dataweave-cookbook.adoc

```diff
@@ -66,6 +66,12 @@ include::partial$dataweave1-links.adoc[tag=dataweave1Examples]
 | xref:dataweave-cookbook-define-a-custom-addition-function.adoc[Define a Custom Addition Function]
 | Defines a function that obtains totals and subtotals through the `accumulator` function. Also performs additions, subtractions and multiplications of numeric values.
 
+| xref:dataweave-cookbook-flatten-arrays.adoc[Flatten Elements of Arrays]
+| Uses `flatten`, the object constructor braces (`{}`), and selectors to modify flatten subarrays and nested elements in arrays.
```

Comment:
Did you mean to choose either "modify" or "flatten" in this: "and selectors to modify flatten subarrays "?